Scripts/Python/xIniDisplay.py

```diff
@@ -62,9 +62,10 @@
 kGraphicsShadows = "Graphics.Shadow.Enable"
 kGraphicsVerticalSync = "Graphics.EnableVSync"
 kGraphicsShadowQuality = "Graphics.Shadow.VisibleDistance"
+kGraphicsDynamicReflections = "Graphics.EnablePlanarReflections"
```

Comment:
For anyone else wondering: this console command has already existed since forever.